storage/watchable_store_test.go

```diff
@@ -222,3 +239,122 @@ func TestUnsafeAddWatching(t *testing.T) {
 		}
 	}
 }
```

Comment:
@xiang90 Updated the test tables as you suggest. And also remove the comment line in watcher_test.go

PTAL.

And thanks!